packages/std/src/results/cosmos_msg.rs

```diff
@@ -19,14 +19,29 @@ where
     // to call into more app-specific code (whatever they define)
     Custom(T),
     Staking(StakingMsg),
+    Wasm(WasmMsg),
     /// A Stargate message encoded the same way as a protobof [Any](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto).
     /// This is the same structure as messages in `TxBody` from [ADR-020](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-020-protobuf-transaction-encoding.md)
     #[cfg(feature = "stargate")]
     Stargate {
         type_url: String,
         data: Binary,
     },
-    Wasm(WasmMsg),
+    /// Sends *native tokens* owned by the contract to the given address on another chain.
```

Comment:
What mean "native token" exactly? A bank token of the contract's chain?

Reply:
Yes. As opposed to a cw20 token.

I will use the term bank token (others in cosmos used native token)